src/cluster_legacy.h

```diff
@@ -117,6 +121,16 @@ typedef struct {
     unsigned char bulk_data[8]; /* 8 bytes just as placeholder. */
 } clusterMsgDataPublish;
 
+typedef struct {
+    uint32_t message_len;
+    unsigned char message_data[8]; /* 8 bytes just as placeholder. */
+} clusterMsgDataPublishMessage;
+
+typedef struct {
+    uint64_t data_count;
+    clusterMsgDataPublishMessage *bulk_data;
+} clusterMsgDataPublishMulti;
+
```

Comment:
Could you place these struct along side the `clusterMsgDataLight` and add a top comment "Light message and it's supported message types"